service/history/queueProcessor.go

```diff
@@ -286,6 +284,31 @@ func (p *queueProcessorBase) processBatch() {
 	return
 }
 
+func (p *queueProcessorBase) submitTask(
```

Comment:
nit: add comment on return value, or change it.
I think when people read code like `submitTask() bool`, people usually thought it mean submit success (true) or fail (false). But this one is a little bit surprising.

Reply:
Makes sense. Reverted the meaning for the return value. Also reverted the return value from `p.taskProcessor.addTask()`